runtime/src/lib.rs

```diff
@@ -598,8 +598,10 @@ impl pallet_domains::Config for Runtime {
 	type DomainsInsertLimit = DomainsInsertLimit;
 	type RegistrationPeriodLimit = RegistrationPeriodLimit;
 	type MaxOuterValueLength = MaxOuterValueLength;
+	type MaxRecordKeySize = ConstU32<256>;
+	type MaxRecordValueSize = ConstU32<4096>;
 	type BaseDomainDeposit = BaseDomainDeposit;
-	type OuterValueByteDeposit = OuterValueByteDeposit;
+	type RecordByteDeposit = TransactionByteFee;
```

Comment:
Looks reasonable to me to use a byte fee that we already have in a runtime.

But due to this change, maybe it's better to rename `TransactionByteFee` in a way to semantically reflect reusability?
@Yung-Beef

Reply:
What do you recommend ?